src/components/SearchTracePage/index.js

```diff
@@ -116,7 +116,7 @@ export default class SearchTracePage extends Component {
             !hasTraceResults && (
               <div className="ui middle aligned center aligned grid" style={{ marginTop: 100 }}>
                 <div className="column">
-                  <img className="js-test-logo" alt="presentation" src={JaegerLogo} width="400" />
+                  <img className="js-test-logo" alt="presentation" src={prefixUrl(JaegerLogo)} width="400" />
```

Comment:
was it using relative url before?

Reply:
No, it uses an absolute URL. 

The image is imported in JS:

https://github.com/jaegertracing/jaeger-ui/blob/7779eb56f485f8d60c2ff158d7a7bfaa9f6252a5/src/components/SearchTracePage/index.js#L25

That does two things:

- Causes the asset to be stored in `/static/media/` with a filename derived from the file's hash
- Stores the eventual absolute path of the media asset in the variable `JaegerLogo`

The absolute path is then used in the JSX as you excerpted. 

The `prefixUrl(...)` adds the path-prefix, if it is configured.